server/controllers/chartsController/index.js

```diff
@@ -1,80 +1,156 @@
-import { ObjectID } from 'mongodb'
-
+import { ObjectId } from 'mongodb'
```

Comment:
I know we've had issues with the casing of this import and the version of the mongodb package in the past. Will this break on staging/production?

Reply:
It shouldn't break anything in production

Reply:
so `mongodb` exports both `ObjectID` and `ObjectId` and they are the same thing?

Reply:
yes they do the same thing. ObjectID is marked as deprecated and they want us to use ObjectId

<img width="542" alt="Screen Shot 2023-01-11 at 4 02 17 PM" src="https://user-images.githubusercontent.com/19805355/211929215-36f7da7e-ff2a-442e-9191-c9f7f449ed48.png">

Reply:
oh great, thanks for the explanation!